docs/user/kafka/confluent.rst

```diff
@@ -7,11 +7,189 @@ Confluent Integration
+e.g. ``"type": ["null","string"]``.
+
+Supported ``SimpleFeature`` Fields
+----------------------------------
+
+To encode types that may be part of a ``SimpleFeature``, but are not part of a standard Avro schema, e.g. ``Geometry``
+or ``Date``, the Confluent Kafka Data Store supports interpreting several key-value metadata attributes on schema
+fields, which are described below.
+
+All the attribute values are case insensitive.
+
+``geomesa.geom.format``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field should be interpreted as a ``Geometry`` in the given format. Must be accompanied by the key
+``geomesa.geom.type``.
+
+=========== ===================== ====================================================
+Value       Schema Field Type     Description
+=========== ===================== ====================================================
+``wkt``     ``STRING``            Well-Known Text representation of a ``Geometry``
+``wkb``     ``BYTES``             Well-Known Binary representation of a ``Geometry``
+=========== ===================== ====================================================
+
+``geomesa.geom.type``
+^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field should be interpreted as a ``Geometry`` of the given type. Must be accompanied by the key
+``geomesa.geom.format``.
+
+======================== ============================
+Value                    Description
+======================== ============================
+``geometry``             A ``Geometry``
+``point``                A ``Point``
+``linestring``           A ``LineString``
+``polygon``              A ``Polygon``
+``multipoint``           A ``MultiPoint``
+``multilinestring``      A ``MultiLineString``
+``multipolygon``         A ``MultiPolygon``
+``geometrycollection``   A ``GeometryCollection``
+======================== ============================
+
+``geomesa.geom.default``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field represents the default ``Geometry`` for this ``SimpleFeatureType``. If the keys
+``geomesa.geom.format`` and ``geomesa.geom.type`` are not present on the same schema field, this attribute will
+be ignored.
+
+=========== ===============================
+Value       Description
+=========== ===============================
+``true``    The default ``Geometry``
+``false``   Not the default ``Geometry``
+=========== ===============================
+
+``geomesa.geom.srid``
+^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field represents a ``[[Geometry]]`` with the given spatial reference identifier (SRID).
+If the keys ``geomesa.geom.format`` and ``geomesa.geom.type`` are not present on the same schema field, this
+attribute will be ignored.
+
+=========== ==============================================================
+Value       Description
+=========== ==============================================================
+``4326``    Longitude/latitude coordinate system on the WGS84 ellipsoid
```

Comment:
Hmm.... does this do anything?  I.e., is there support for reprojecting to 4326 or doing something sane here?  (If not, it may be worth dropping it.)

Reply:
Some of the SFTs that I wanted to use this for have a property `srid = 4326`. I don't know what it does but I just wanted to make it completely compatible with them.

Reply:
Removed